Fix Interns not walking when OrderToStopSprint is called

T-Rizzle12 opened the pull request:
I noticed a small bug when making my own personal changes to the mod is that OrderToStopSprint has a logic error in it that causes the bots to not stop sprinting when called
```C#
/// <summary>
/// Set the controller to stop sprinting
/// </summary>
public void OrderToStopSprint()
{
    if (Npc.inSpecialInteractAnimation || !IsTouchingGround || Npc.isClimbingLadder)
    {
        return;
    }
    if (this.IsJumping)
    {
        return;
    }
    // This was Npc.isSprinting, which essentially meant only stop sprinting while not sprinting
    // I added a ! in front which fixed the issue.
    if (!Npc.isSprinting)
     {
         return;
     }

    floatSprint = 0f;
}
```
Changes in this pull request:
- Fix bug in OrderToStopSprint where the bot would only stop sprinting while not sprinting. "Yeah, it's confusing"

Szumi57 commented:
Oh thanks, I will merge the commit.
For info, for next update, there will be a lot of changes to the AI, by using behavior trees instead of states, so expect a lot of code removed (if you're still tinkering stuff)

T-Rizzle12 commented:
@Szumi57 
I have rewritten so much of the addon, it’s practically a new mod. There is still some of the same old code, particularly around the player and enemy patches, but I have added a lot to it. I’m excited to see how these behavior trees work out. I may end up changing from the state system as well.
